src/tests/token/test_erc20.cairo

```diff
@@ -147,7 +147,7 @@ fn test__approve_from_zero() {
 fn test__approve_to_zero() {
     let mut state = setup();
     testing::set_caller_address(OWNER());
-    state.erc20._approve(OWNER(), Zeroable::zero(), VALUE);
+    state.erc20._approve(OWNER(), ZERO(), VALUE);
```

Comment:
why do we have `ZERO()` instead of using `Zeroable::zero()`?

Reply:
IMO shorter and easier to read and follow (is not the same to ctrl+f Zeroable::zero that is used for everything zeroable, than to look for the ZERO() constant)

Reply:
we could simply use `zero()` from `Zeroable` if it's a matte of length. I don't see much value in it and the fact that is used for everything zero is not a problem. Otoh I don't think it's that important, there's no problem with `ZERO()` either.